### PR TITLE
chore: prepare 4.7.0-emqx-1 version + appup instructions

### PR DESCRIPTION
**Pull request by keynslug**
[EMQX-8562](https://emqx.atlassian.net/browse/EMQX-8562)

### CODE REVIEW

**Review by zmstone — APPROVED** [review]

File: src/epgsql.app.src [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {application, epgsql,
  [{description, "PostgreSQL Client"},
-  {vsn, "4.6.1"},
+  {vsn, "4.7.0-emqx-1"},
```
Comment (zmstone) [inline src/epgsql.app.src:3]:
```suggestion
  {vsn, "4.7.0.1"},
```

**Comment by zmstone** [comment]
sorry, approved too fast. the -emqx-1 is perhaps going to cause some troubles in release handler (which may complain that it's not a semver or something, cant recall the exact error).

File: src/epgsql.app.src (hunk 1 above)
Comment (keynslug) [reply src/epgsql.app.src:3]: Oops. Will open a followup PR shortly.

Comment (keynslug) [reply src/epgsql.app.src:3]: #8

**Pull request merged**